pyinjective/composer.py

```diff
@@ -46,61 +50,256 @@
-    "MsgCreateBinaryOptionsLimitOrder": injective_exchange_tx_pb.MsgCreateBinaryOptionsLimitOrderResponse,
-    "MsgCreateBinaryOptionsMarketOrder": injective_exchange_tx_pb.MsgCreateBinaryOptionsMarketOrderResponse,
-    "MsgCancelBinaryOptionsOrder": injective_exchange_tx_pb.MsgCancelBinaryOptionsOrderResponse,
-    "MsgAdminUpdateBinaryOptionsMarket": injective_exchange_tx_pb.MsgAdminUpdateBinaryOptionsMarketResponse,
-    "MsgInstantBinaryOptionsMarketLaunch": injective_exchange_tx_pb.MsgInstantBinaryOptionsMarketLaunchResponse,
+    "MsgCreateSpotLimitOrder":
+        injective_exchange_tx_v2_pb.MsgCreateSpotLimitOrderResponse,
+    "MsgCreateSpotMarketOrder":
+        injective_exchange_tx_v2_pb.MsgCreateSpotMarketOrderResponse,
+    "MsgCreateDerivativeLimitOrder":
+        injective_exchange_tx_v2_pb.MsgCreateDerivativeLimitOrderResponse,
+    "MsgCreateDerivativeMarketOrder":
+        injective_exchange_tx_v2_pb.MsgCreateDerivativeMarketOrderResponse,
+    "MsgCancelSpotOrder":
+        injective_exchange_tx_v2_pb.MsgCancelSpotOrderResponse,
+    "MsgCancelDerivativeOrder":
+        injective_exchange_tx_v2_pb.MsgCancelDerivativeOrderResponse,
+    "MsgBatchCancelSpotOrders":
+        injective_exchange_tx_v2_pb.MsgBatchCancelSpotOrdersResponse,
+    "MsgBatchCancelDerivativeOrders":
+        injective_exchange_tx_v2_pb.MsgBatchCancelDerivativeOrdersResponse,
+    "MsgBatchCreateSpotLimitOrders":
+        injective_exchange_tx_v2_pb.MsgBatchCreateSpotLimitOrdersResponse,
+    "MsgBatchCreateDerivativeLimitOrders":
+        injective_exchange_tx_v2_pb.MsgBatchCreateDerivativeLimitOrdersResponse,
+    "MsgBatchUpdateOrders":
+        injective_exchange_tx_v2_pb.MsgBatchUpdateOrdersResponse,
+    "MsgDeposit":
+        injective_exchange_tx_v2_pb.MsgDepositResponse,
+    "MsgWithdraw":
+        injective_exchange_tx_v2_pb.MsgWithdrawResponse,
+    "MsgSubaccountTransfer":
+        injective_exchange_tx_v2_pb.MsgSubaccountTransferResponse,
+    "MsgLiquidatePosition":
+        injective_exchange_tx_v2_pb.MsgLiquidatePositionResponse,
+    "MsgIncreasePositionMargin":
+        injective_exchange_tx_v2_pb.MsgIncreasePositionMarginResponse,
+    "MsgCreateBinaryOptionsLimitOrder":
+        injective_exchange_tx_v2_pb.MsgCreateBinaryOptionsLimitOrderResponse,
+    "MsgCreateBinaryOptionsMarketOrder":
+        injective_exchange_tx_v2_pb.MsgCreateBinaryOptionsMarketOrderResponse,
+    "MsgCancelBinaryOptionsOrder":
+        injective_exchange_tx_v2_pb.MsgCancelBinaryOptionsOrderResponse,
+    "MsgAdminUpdateBinaryOptionsMarket":
+        injective_exchange_tx_v2_pb.MsgAdminUpdateBinaryOptionsMarketResponse,
+    "MsgInstantBinaryOptionsMarketLaunch":
+        injective_exchange_tx_v2_pb.MsgInstantBinaryOptionsMarketLaunchResponse,
 }
 
 GRPC_MESSAGE_TYPE_TO_CLASS_MAP = {
-    "/injective.exchange.v1beta1.MsgCreateSpotLimitOrder": injective_exchange_tx_pb.MsgCreateSpotLimitOrder,
```

Comment:
same refactoring suggestion as for the map above. replace hardcoding with pattern matching